modules/con_notifications-configuration.adoc

```diff
@@ -98,23 +98,22 @@ spec:
 apiVersion: argoproj.io/v1alpha1
 kind: NotificationsConfiguration
 metadata:
- name: default-notifications-configuration # <1>
+ name: default-notifications-configuration #<1>
 spec:
- subscriptions: |
-  subscriptions: | # <2>
+ subscriptions: #<2>
```

Comment:
subscriptions.subscriptions is intended?

Reply:
Hey @dfitzmau, this is intended to configure notification controller.

Reply:
Hi @dfitzmau, are you satisfied with this resolution and can I close the comment? I have squashed the commit and if you are fine with the changes, I will move this ticket for merge-review.

Reply:
Yes. PLease proceed with merging.